src/teacher.rb

```diff
@@ -0,0 +1,17 @@
+require './person'
+
+class Teacher < Person
+  def initialize(
+    age:,
+    specialization:,
+    name: 'undefined',
+    parent_permission: false
+  )
+    super(name, parent_permission, age)
```

Comment:
- [ ] Kindly ensure you are mindful of the parameters and their arrangements. `age` should come before `name` and `parent_permisson` in the `#super` method.

- [ ] Since you are using the colon notation the assignment of parameters will differ also

  ```
  super(age: age, name: name, parent_permission: parent_permission)
  ```

Reply:
🔥 Fixed